packages/projection/README.md

```diff
@@ -1,20 +1,100 @@
 # Cardano JS SDK | projection
 
-Chain Sync event projection utilities.
+This package is a library of generic projection types and utilities.
```

Comment:
Wouldn't it be nice if we explained what 'projecting' or 'projection' means in our context? (in a relevant README if not here)

Reply:
Just link this @mkazlauskas https://event-driven.io/en/projections_and_read_models_in_event_driven_architecture/

Reply:
Done in https://github.com/input-output-hk/cardano-js-sdk/pull/952/commits/e8d9e3ac77b682ce213d9fc817c72f56893acf14